gradle.properties

```diff
@@ -1,19 +1,19 @@
-GROUP=com.squareup.inject
+GROUP=app.cash.inject
 VERSION_NAME=1.0.0-SNAPSHOT
 
 POM_DESCRIPTION=Constructor-inject views during layout inflation
 
-POM_URL=https://github.com/square/InflationInjection/
-POM_SCM_URL=https://github.com/square/InflationInjection/
-POM_SCM_CONNECTION=scm:git:git://github.com/square/InflationInjection.git
-POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/square/InflationInjection.git
+POM_URL=https://github.com/cashapp/InflationInjection/
+POM_SCM_URL=https://github.com/cashapp/InflationInjection/
+POM_SCM_CONNECTION=scm:git:git://github.com/cashapp/InflationInjection.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/cashapp/InflationInjection.git
 
 POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 
-POM_DEVELOPER_ID=square
-POM_DEVELOPER_NAME=Square, Inc.
+POM_DEVELOPER_ID=cashapp
+POM_DEVELOPER_NAME=CashApp
```

Comment:
Is this what we use elsewhere? Without the space?

Reply:
No idea.

SQLDelight DGAF
https://github.com/cashapp/sqldelight/blob/90204d0d047075b34395c4b49a708f8379b40698/gradle.properties#L13-L14

Turbine (also by me):
https://github.com/cashapp/turbine/blob/a710b59dc8010e952611a5c2ff0882e9311b63f9/gradle.properties#L20-L21

Are you saying it needs more ™™™™™?

Reply:
We need to decide now or it's going to get cargo culted to all hell...

Reply:
It can be changed at any time so... merging.